org.beeref.BeeRef.json

```
@@ -0,0 +1,56 @@
+{
+    "id": "org.beeref.BeeRef",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "21.08",
+    "sdk": "org.freedesktop.Sdk",
```

```suggestion
    "runtime": "org.kde.Platform",
    "runtime-version": "6.3",
    "sdk": "org.kde.Sdk",
```

Otherwise you don't have Qt.